schemas/Templatecontent.json

```diff
@@ -0,0 +1,76 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+      "Templatecontent": {
+        "$ref": "#/definitions/Templatecontent"
+      }
+    },
+    "required": [
+      "Templatecontent"
+    ],
+    "title": "Templatecontent",
+    "definitions": {
+      "Templatecontent": {
+        "$id": "#/properties/Templatecontent",
+        "type": "object",
+        "title": "The Template content schema",
+        "description": "All Template content.",
+        "required": [],
+        "properties": {
+          "template_id": {
+            "$id": "#/properties/template_id",
+            "type": "string"
+          },
+          "state": {
+            "$id": "#/properties/state",
+            "type": "string"
+          },
+          "language": {
+            "$id": "#/properties/language",
+            "type": "string"
+          },
+          "subject": {
+            "$id": "#/properties/subject",
+            "type": "string"
+          },
+          "body": {
+            "$id": "#/properties/body",
+            "type": "string"
+          },
+          "replacement_tags": {
+            "$id": "#/properties/replacement_tags",
+            "type": "string"
```

Comment:
Is this correct ?

Reply:
changed this to array. We will store array of objects